packages/next/src/build/webpack/loaders/next-image-loader/blur.ts

```diff
@@ -58,7 +60,10 @@ export async function getBlurImage(
       // because it can delay starting the dev server. Instead, we inline a
       // special url to lazily generate the blur placeholder at request time.
       const prefix = 'http://localhost'
-      const url = new URL(`${basePath || ''}/_next/image`, prefix)
+      const url = new URL(
+        `${basePath || ''}${nextUrlServerPrefix || ''}/_next/image`,
```

Comment:
I feel like this could be a flag under images config that says `useAssetPrefix` or similar. I feel like adding a new config for a prefix could cause further confusion between which config to use when which we already see with `basePath` and `assetPrefix`.